wasm_utils/src/macros.rs

```diff
@@ -26,7 +26,7 @@ macro_rules! ribosome_success {
 macro_rules! ribosome_error_code {
     ($s:ident) => {
         Ok(Some(RuntimeValue::I32(
-            ::holochain_wasm_utils::holochain_core_types::error::RibosomeErrorCode::$s as i32,
+            $crate::holochain_core_types::error::RibosomeErrorCode::$s as i32,
```

Comment:
Also, are you sure this works in all contexts?

Reply:
sure, why wouldn't it?